### PR TITLE
Features/remove lease ups from gen listings #157445311

### PR DESCRIPTION
**Pull request by chrisdolendo**

### CODE REVIEW

**Review by akegan — APPROVED** [review]
Looks good! nice quick fix 😄

**Comment by akegan** [comment]
@chrisdolendo this has been product accepted but has merge conflicts 😢

File: app/services/force/listing_service.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ class ListingService < Force::Base
     def listings
       parsed_index_query(%(
         SELECT #{query_fields(:index)} FROM Listing__c
-        WHERE #{user_can_access}
+        WHERE Status__c != 'Lease Up'
```
Comment (chrisdolendo) [inline app/services/force/listing_service.rb:13]: @akegan just checking to make sure - master doesn't have the WHERE #{user_can_access} line anymore to allow more viewing permsissions correct? This PR included which is why there was a merge conflict. I updated this logic to just say WHERE Status__c != 'Lease Up' and then I'll merge to master, but wanted to make sure first.

Comment (chrisdolendo) [reply app/services/force/listing_service.rb:13]: @akegan just checking on this before I merge it!

Comment (akegan) [reply app/services/force/listing_service.rb:13]: yep we're good to remove it! the function should be gone too 🤔

Comment (akegan) [reply app/services/force/listing_service.rb:13]: looks the the function is also gone so good to go!

**Pull request merged**